packages/kbn-securitysolution-es-utils/.babelrc

```
@@ -0,0 +1,3 @@
+{
+  "presets": ["@kbn/babel-preset/node_preset"]
```

```suggestion
  "presets": ["@kbn/babel-preset/node_preset"],
  "ignore": ["**/*.test.ts", "**/*.test.tsx"]
```

I think you want the ignore here like the others